Samples/WebSignIn/README.md

```diff
@@ -0,0 +1,30 @@
+# WebSignIn Samples
+
+This sample contains three separate sample application schemes:
```

Comment:
Same comments on abstract and schemes as above. I also can't make a suggestion for the abstract as I don't know enough to see the common thread for all of these schemes. I think they're all using web-based sign-in, but I thought that was what the point of that portion of the mobile SDK, to support the redirect model by using the sign-in widget.

Either way, this really needs to be an abstract of what this does. It may be that these samples aren't related and need to be reorganized.

If they are related, then I'd recommend something like:

## Abstract
(what they do)

## Overview

(any more required background and an intro to what they are followed by a table with each sample and the abstract fro that sample... similar to my suggestions for the table in the shared Readme file.)

> **NOTE:**
> This means that each of the of the three samples should have it's own Readme as well. The rest of the suggestions assume that each one is it's own readme.